cygnus-twitter/pom.xml

```diff
@@ -0,0 +1,242 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.telefonica.iot</groupId>
+    <artifactId>cygnus-twitter</artifactId>
+    <version>0.1.0_SNAPSHOT</version>
```

Comment:
All the agent versions must be the same, independently of the moment the agent was added to the repo. This is because upon releasing a single git release can be produced.